Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,26 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+Exercise 1
+a) O(n)
+b) O(n^n)
+c) O(n^2)
+
```

Comment:
c) this is a recursive loop that's going to start at whatever value bunnies starts out as is and run until bunnies -1 is 0, so it's linear, O(n)